src/Trains.Emoji/Program.cs

```diff
@@ -10,80 +11,141 @@ namespace Trains.Emoji
 {
     public class EmojiDrawer
     {
+        private static int s_numberOfTrainsToDraw = 6;
+        private static int s_numberOfTreesToDraw = 3;
+
         private readonly IRenderer<Tree> _tree;
         private readonly IRenderer<Track> _track;
+        private readonly IRenderer<Track> _bridge;
         private readonly IRenderer<Train> _train;
         private readonly IPixelMapper _pixelMapper;
-        private const string FolderName = "EmojiOutput";
+        private const string BaseFolderName = "EmojiOutput";
 
-        public static void Main()
-            => DI.ServiceLocator.GetService<EmojiDrawer>().Save(512);
+        public static void Main(string[] args)
+        {
+            var imageSizes = new List<int>();
 
-        public EmojiDrawer(IRenderer<Tree> tree, IRenderer<Track> track, IRenderer<Train> train, IPixelMapper pixelMapper)
+            for (int i = 0; i < args.Length; i++)
```

Comment:
Wasn't there a simple MS library for this? More of an open question!

Reply:
Yes, there is. I think upgrading to it makes for good stream content :)